homeassistant/components/fan/__init__.py

```diff
@@ -272,15 +272,21 @@ async def async_set_percentage(self, percentage: int) -> None:
         else:
             await self.async_set_speed(self.percentage_to_speed(percentage))
 
-    async def async_increase_speed(self, percentage_step=None) -> None:
+    async def async_increase_speed(
+        self, percentage_step: Optional[float] = None
```

Comment:
The percentage step is validated to an integer in the service schema.

Reply:
Right on, I'm thinking of the first version I did of this before I changed it.  Little too much task switching this week.